app/models/loan.rb

```diff
@@ -11,7 +11,7 @@ class Loan < ActiveRecord::Base
     end
   end
 
-  scope :active, -> { where(closed: false) }
+  scope :active, -> { where(closed: false).joins(:game).where('games.culled' => false) }
```

Comment:
Good catch. We can merge this provided the above publisher search stuff is removed, but for now I'm gonna close this.